rate.go

```diff
@@ -102,3 +131,22 @@ func (l *Limiter) incr(name string, dur time.Duration) (int64, error) {
 	rate, _ := incr.Result()
 	return rate, err
 }
+
+func (l *Limiter) get(name string) (int64, error) {
```

Comment:
I think instead of get you should modify `incr` to accept `n int64` and use it as `incr(name, dur, 0)` to get value without incrementing it.